lib/sync-records.ts

```diff
@@ -183,8 +183,10 @@ export const syncWorkbookRecords = async ({
     await upsertEmployeesWithManager();
 
     let retriesAttempted = 0;
-    const retryUpsert = async (retriesScheduled: number) => {
-      while (retriesAttempted < retriesScheduled) {
+    const MAX_RETRY_ATTEMPTS = 2;
+
+    const retryUpsert = async (MAX_RETRY_ATTEMPTS: number) => {
```

Comment:
Should this function parameter be lowercase? Would read better and might have weird var shadowing bugs for not realizing which one you're using

Reply:
Edited